cmd/bridge/main.go

```diff
@@ -7,25 +7,21 @@ import (
 	"flag"
 	"fmt"
 	"io/ioutil"
+	"k8s.io/klog"
+	"log"
```

Comment:
I dont think you want `log`.  From what I can tell, its a new import here, which is only because you are adding those extra helpers below.

```golang
# needed?
log.SetOutput(KlogWriter{})
log.SetFlags(0)
```

Reply:
I will try it without defining a klog writer but all the examples I saw did this. Let me try.

Reply:
I think you may be looking at some more complex integration examples then.  Since you are removing `capnslog` and replacing with `klog`, it is as if its a clean install, there is nothing fancy needed.